portfolio/src/main/webapp/main_page_clickables/game.html

```diff
@@ -0,0 +1,17 @@
+<p>
+    So this is how it works: you will click this 'Random Country' button below, and... yes, a random country will appear. 
```

Comment:
I think you should revisit this after completing this weeks assignment -- you'll know  more tools for making this more sophisticated.

Some suggestions on what to look out for:
 - Load the random country name and insert it into the page -- you've got this working.
 - Create a `<form>` with two elements
   - an editable `<input ..>` for the user to enter the capital
   - a hidden `<input ...>` for the country name (so we can send it back to the servlet together with the guess)
   - set the action to post back to `/country` (or `/data`, in the current version)
- In the servlet, handle the Post request
  - You're getting user data -- you need to validate it carefully because we don't want to trust the client data (ever!)
  - See if the `country` value in the post request exists, and is a valid country
  - See if the `capitalName` value matches the capital for the country
  - If this matches, increment `correct++` else `wrong++` guesses
- In the response, send back the result (right/wrong, the current score (# of right/wrong), and the next country to guess.
- Update the fetch function to handle multiple values.

The next steps in this week will go into `<form>` posting, returning structured data, etc.. so keep your game in mind as you learn about those